src/drawing_utilities.ts

```diff
@@ -1,93 +1,109 @@
 /* 
-This file is designed to hold all the functions 
-required for drawing to the canvas
+This file is designed to hold helper functions for drawing to the canvas
 */
 
 import { ULabelAnnotation } from './annotation';
 import { VALID_HTML_COLORS } from "./colors";
 
-/* Receives a base color and applies a gradient to it based on its confidence
-The reason for passing in the get confidence function is so that we can apply
-a gradient based on diffrent confidence statistics if we choose to do so. */
-export function apply_gradient(
+/**
+ * Applies a linear gradient to a color channel based on its confience.
+ * 
+ * @param {number} color_channel Color channel
```

Comment:
Maybe wrap this in a function that also gets the color_channel value from a hexstring